server/test/services/applications/ProgramAdminApplicationServiceTest.java

```diff
@@ -105,4 +164,192 @@ public void getNote_multipleNotes_findsLatest() throws Exception {
     // Execute, verify.
     assertThat(service.getNote(application)).contains(note);
   }
+
+  @Test
+  public void setStatus_sendsEmail() throws Exception {
+    String userEmail = "user@email.com";
+    SimpleEmail simpleEmail = Mockito.mock(SimpleEmail.class);
+    String programName = "some-program";
+    service =
+        new ProgramAdminApplicationService(
+            instanceOf(ApplicantService.class),
+            instanceOf(ApplicationRepository.class),
+            instanceOf(ApplicationEventRepository.class),
+            instanceOf(Config.class),
+            simpleEmail);
+
+    ProgramDefinition program =
+        ProgramBuilder.newActiveProgram("some-program")
+            .withStatusDefinitions(new StatusDefinitions(ORIGINAL_STATUSES))
+            .buildDefinition();
+    Account account = resourceCreator.insertAccount();
+    Applicant applicant = resourceCreator.insertApplicantWithAccount(Optional.of(userEmail));
+    Application application =
+        Application.create(applicant, program.toProgram(), LifecycleStage.ACTIVE)
+            .setSubmitTimeToNow();
+
+    StatusEvent event =
```

Comment:
Worth adding an additional assertion that the latest status event log matches this?